pkg/provider/gitlab/gitlab.go

```diff
@@ -226,6 +247,31 @@ func (g *Gitlab) GetSecretMap(ctx context.Context, ref esv1beta1.ExternalSecretD
 	return secretData, nil
 }
 
+func isValidSecret(checkName bool, ref esv1beta1.ExternalSecretFind, environment string, data *gitlab.ProjectVariable) (bool, string) {
```

Comment:
at first sight, i was irretated by the name "isValid". what should be validated in case of "getAllSecretes" ?

Reply:
renamed to "isFilteredSecret".